src/data/adventures.txt

```diff
@@ -220,11 +220,11 @@ Island	adventure=26	DiffLevel: mid Env: outdoor Stat: 30	Hippy Camp	outfit|1 win
 Island	adventure=65	DiffLevel: mid Env: outdoor Stat: 30	Hippy Camp (Hippy Disguise)
 
 Island	adventure=149	DiffLevel: high Env: outdoor Stat: 100	The Hippy Camp (Bombed Back to the Stone Age)	+5 figurine of a stinking seal
-Island	adventure=150	DiffLevel: unknown Env: outdoor Stat: 100	The Orcish Frat House (Bombed Back to the Stone Age)	+5 figurine of a slippery seal
+Island	adventure=150	DiffLevel: high Env: outdoor Stat: 100	The Orcish Frat House (Bombed Back to the Stone Age)	+5 figurine of a slippery seal
 
 Island	adventure=66	DiffLevel: unknown Env: outdoor Stat: 45	The Obligatory Pirate's Cove	outfit|+1 bottle of pirate juice
 
-Pirate	adventure=157	DiffLevel: unknown Env: indoor Stat: 80	Barrrney's Barrr	Cap'm Caronch's Map|Orcish Frat House blueprints|6 pirate insults|7 pirate insults
```

Comment:
Whoah. When I tried to derive this earlier, using the Autumn-aton with either the Swashbuckling Getup or pirate fledges redirected to the cove - the container document with the pirate ship.

I reported the bug, and apparently KoL fixed it.

Reply:
No, it still happens for me. How did you do this? Using the command?

Reply:
Yeah, `autumnaton send barrr`.